com.endlessnetwork.fablemaker.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id" : "com.endlessnetwork.fablemaker",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "18.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "build-options" : {
+    "strip" : true,
+    "no-debuginfo" : true
+  },
+  "command" : "com.endlessnetwork.fablemaker.sh",
+  "modules" : [
+    { 
+        "name" : "Fablemaker",
+        "buildsystem" : "simple",
+        "only-arches" : ["x86_64"],
+	"sources" : 
```

Comment:
can you fix the indenting on this file? it's a little hard to read. run it through a linter like jq